src/licensedcode/models.py

```diff
@@ -768,6 +768,63 @@ def get_rules(
     licenses_as_rules = build_rules_from_licenses(licenses_db)
     return chain(licenses_as_rules, rules)
 
```

Comment:
Nitpick: include another new line here. There should be two new lines between functions, as per the PEP 8 style guide.